CHANGELOG.md

```diff
@@ -2,6 +2,35 @@
 
 Full documentation for rocDecode is available at [https://rocm.docs.amd.com/projects/rocDecode/en/latest/](https://rocm.docs.amd.com/projects/rocDecode/en/latest/)
 
+## rocDecode 0.9.0 (unreleased)
+
+### Changed
+
+* AMD Clang++ is now the default CXX compiler.
+* `rocDecode-setup.py` setup script updates to common package install
```

Comment:
This needs more details. How did it behave before; how does it behave now; how does this affect the end user.

Reply:
updated